scripts/ci/perf_test.sh

```diff
@@ -44,7 +44,7 @@ done
 trap "aws --region us-west-2 ec2 terminate-instances --instance-ids $ec2_instance_id" KILL
 
 # push binary and script to instance
-debname=$(find /tmp/workspace/artifacts/influxdb2-server*amd64.deb)
+debname=$(find /tmp/workspace/artifacts/influxdb2*amd64.deb)
```

Comment:
This one was formerly `influxdb*amd64.deb`. Is leaving the `2` here okay/intended?

Reply:
Not intended but should be ok since the prefix of the deb is still `influxdb2`